### ISSUE TITLE
ethereum wallet not connecting with geth/node

### ISSUE THREAD

**Issue by Stev4789** [ethereum/mist#4159]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.11.1`
OS & Version: windows/linux/osx
Node version: `geth 1.8.20`
Number of blocks synchronized: 6,900,000
```
[2018-12-20T00:04:40.257] [INFO] Settings - Running in production mode: true
[2018-12-20T00:04:40.305] [INFO] EthereumNode - undefined 'fast' 'light'
[2018-12-20T00:04:40.307] [INFO] EthereumNode - Defaults loaded: geth main fast
[2018-12-20T00:04:40.497] [INFO] main - Starting in Wallet mode
[2018-12-20T00:04:40.519] [INFO] Db - Loading db: C:\Users\steven\AppData\Roaming\Ethereum Wallet\mist.lokidb
[2018-12-20T00:04:40.529] [INFO] Windows - Creating commonly-used windows
[2018-12-20T00:04:40.531] [INFO] Windows - Create secondary window: loading, owner: notset
[2018-12-20T00:04:40.604] [INFO] updateChecker - Check for update...
[2018-12-20T00:04:45.900] [INFO] Sockets/node-ipc - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2018-12-20T00:04:47.000] [INFO] Windows - Create primary window: main, owner: notset
[2018-12-20T00:04:47.022] [INFO] ClientBinaryManager - Initializing...
[2018-12-20T00:04:47.023] [INFO] ClientBinaryManager - Checking for new client binaries config from: https://raw.githubusercontent.com/ethereum/mist/master/clientBinaries.json
[2018-12-20T00:04:47.047] [INFO] main - Loading Interface at file://C:\Program Files\Ethereum-Wallet\resources\app.asar/interface/wallet/index.html
[2018-12-20T00:04:47.101] [INFO] Sockets/node-ipc - Connected!
[2018-12-20T00:04:47.548] [INFO] Sockets/3 - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2018-12-20T00:04:47.549] [INFO] Sockets/3 - Connected!
[2018-12-20T00:04:48.259] [WARN] ClientBinaryManager - Error loading local config - assuming this is a first run: Error: ENOENT: no such file or directory, open 'C:\Users\steven\AppData\Roaming\Ethereum Wallet\clientBinaries.json'
[2018-12-20T00:04:48.259] [INFO] ClientBinaryManager - Write new client binaries local config to disk ...
[2018-12-20T00:04:48.261] [INFO] ClientBinaryManager - No "skippedNodeVersion.json" found.
[2018-12-20T00:04:48.262] [INFO] ClientBinaryManager - Initializing...
[2018-12-20T00:04:48.262] [INFO] ClientBinaryManager - Resolving platform...
[2018-12-20T00:04:48.263] [INFO] ClientBinaryManager - Calculating possible clients...
[2018-12-20T00:04:48.264] [INFO] ClientBinaryManager - 1 possible clients.
[2018-12-20T00:04:48.265] [INFO] ClientBinaryManager - Verifying status of all 1 possible clients...
[2018-12-20T00:04:48.265] [INFO] ClientBinaryManager - Verify Geth status ...
[2018-12-20T00:04:48.440] [INFO] updateChecker - App is up-to-date.
[2018-12-20T00:04:48.493] [INFO] ClientBinaryManager - Checking for Geth sanity check ...
[2018-12-20T00:04:48.493] [INFO] ClientBinaryManager - Checking for Geth sanity check ...
[2018-12-20T00:04:48.493] [INFO] ClientBinaryManager - Checking sanity for Geth ...
[2018-12-20T00:04:48.496] [INFO] ClientBinaryManager - Checking sanity for Geth ...
[2018-12-20T00:04:48.726] [ERROR] ClientBinaryManager - Sanity check failed for Geth Error: Unable to find "1.8.16" in Geth output
    at Promise.resolve.then.then.then (C:\Program Files\Ethereum-Wallet\resources\app.asar\node_modules\ethereum-client-binaries\src\index.js:632:17)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2018-12-20T00:04:48.755] [ERROR] ClientBinaryManager - Sanity check failed for Geth Error: Unable to find "1.8.16" in Geth output
    at Promise.resolve.then.then.then (C:\Program Files\Ethereum-Wallet\resources\app.asar\node_modules\ethereum-client-binaries\src\index.js:632:17)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2018-12-20T00:04:48.758] [INFO] ClientBinaryManager - Download binary for Geth ...
[2018-12-20T00:04:48.759] [INFO] ClientBinaryManager - Downloading package from https://gethstore.blob.core.windows.net/builds/geth-windows-amd64-1.8.16-477eb093.zip to C:\Users\steven\AppData\Roaming\Ethereum Wallet\binaries\Geth\archive.zip ...
[2018-12-20T00:04:50.012] [INFO] (ui: popupWindow) - Meteor starting up...
[2018-12-20T00:04:51.323] [INFO] Sockets/2 - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2018-12-20T00:04:51.323] [INFO] Sockets/2 - Connected!

Basically I downloaded the newest version of geth, synced it then tried to open the wallet. The wallet then connects to a remote node instead of the local node; or I get the blue circling arrow; or the waiting on block error message.

**Comment by Stev4789** [comment]
tried to start new wallet with geth 1.8.20 ..... solution It just does not work with 1.8.20 had to delete chain-state and restart from beginning with geth1.8.16
